eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml

```diff
@@ -0,0 +1,14 @@
+eight_points_guzzle:
+    clients:
+        api_payment:
+            base_url: "http://api.domain.tld"
+
+            options:
+                timeout: 30
+
+                headers:
+                    Accept: "application/json"
```

Comment:
Add a comment before this line to explain what we are doing. 
Maybe aleo change this to an less important header. Like the User-Agent header.

Reply:
wrote comment